Ds3/Helpers/Ds3Diagnostics/Ds3Diagnostic.cs

```diff
@@ -0,0 +1,104 @@
+﻿/*
+ * ******************************************************************************
+ *   Copyright 2014-2016 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+using System.Collections.Generic;
+using System.Linq;
+using Ds3.Models;
+
+namespace Ds3.Helpers.Ds3Diagnostics
+{
+    /// <summary>
+    /// Diagnostic helper class
+    /// </summary>
+    public class Ds3Diagnostic
+    {
+        private IDs3Client Client { get; }
+
+        /// <summary>
+        /// Gets the <see cref="CacheFilesystemInformation"/> for all cache that are near capacity.
+        /// A cache is determined near capacity if the current utilization is at or exceeds <see cref="CacheNearCapacity.CacheUtilizationNearCapacityLevel"/>.
+        /// If no file systems are near capacity, than an empty collection is returned.
+        /// </summary>
+        public IEnumerable<CacheFilesystemInformation> CacheFilesystemInformations { get; private set; }
```

Comment:
Information does not need an added s at the end.  Information is already plural.